Documentation/cli/README.md

```diff
@@ -32,6 +32,7 @@ Command | Description
 [print](#print) | Evaluate an expression.
 [regs](#regs) | Print contents of CPU registers.
 [restart](#restart) | Restart process from a checkpoint or event.
+[reverse-stepi](#reverse-stepi) | Reverse single step a single cpu instruction.
```

Comment:
We don't have a `stepi` command, it's called `step-instruction` so it's weird that this is `reverse-stepi`.

Reply:
Ah I see.  I can rename it to reverse-step-instruction.

Reply:
Maybe we should make the prefix be `rev-` instead of reverse and have `rev-step-instruction`, `rev-next` and `rev-step`. Alternatively make `reverse`/`rev` be a prefix, like `on` and `deferred` and have `rev step-instruction`, `rev next` and `rev step`. cc @derekparker

Reply:
I like the approach to have a rev prefix and act as a modifier.  Once it is agreed on, I will rework my patch and update the PR.

Reply:
@aarzilli +1, I like the prefix approach a lot, `rev [command]` works for me.